do.py

```diff
@@ -459,6 +460,42 @@ def getstatusoutput(command):
     )
 
 
+def build(clean_env="false", sdk="all"):
```

Comment:
sdk should be the first argument as that's more frequently going to be used.

clean_env accepting false/true is not intuitive; maybe use `env_setup` with possible values as None and "clean"; we might add more later (e.g. reinstall_deps, etc.)

Reply:
done